pydantic/types.py

```diff
@@ -2411,6 +2411,8 @@ def get_json_format(cls) -> Literal['base64url']:
         return 'base64url'
 
 
+# TODO: align with PastDate and other similar types to enable usage alone
+# and add an if TYPE_CHECKING block with Annotated[bytes, ...] etc to make type checkers happy
```

Comment:
Following up - I don't think this makes sense, as `EncodedBytes` takes an encoder argument, so it doesn't really make sense to initialize independently, I think we can leave the structure as is.